.github/workflows/makefile.yml

```diff
@@ -12,7 +12,8 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-
+    permissions:
+      contents: write
```

Comment:
This shouldn't be needed, as I have it at a global level above on lines 3-4.  Please remove so I can merge this PR.

Reply:
Not sure how this got in here. Must have slipped in my local copy when attempting to test the permissions issues we had with the GitHub Action. It's corrected in the revised commit.